core/src/main/java/com/ritense/valtimo/domain/user/UserSettings.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ *  Licensed under EUPL, Version 1.2 (the "License");
```

Comment:
```suggestion
 * Licensed under EUPL, Version 1.2 (the "License");
```

The checkstyle is failing with an error because there is 1 space too many. Error:
`UserSettingsService.java:4: Line does not match expected header line of '^ \* Licensed under EUPL, Version 1.2 \(the "License"\);$'. [RegexpHeader]`